README.md

```diff
@@ -1 +1,63 @@
-# Charlie backend
\ No newline at end of file
+# WHAT backend
+
+## 1. About the project
+
+The “WHAT” project (which is absent today) is intended to manage the educational process in educational centers where the student-mentor system operates. The service allows you to track progress, attendance, materials studied, as well as view lists of students, mentors and other useful information about the educational process.
+
+## 2. Where to find UI part of the project
+
+Here is the UI part of project "WHAT": https://github.com/ita-social-projects/what-front
+
+---
+
+## Table of Contents
+
+- [Documentation](#Documentation)
+  - [Required to install](#Required-to-install)
+  - [Setup](#Setup)
+  - [How to run local](#How-to-run-local)
+  - [Git flow](#git-flow)
+- [FAQ](#faq)
+- [License](#license)
+
+---
+
+## 3. Documentation
+
+### Required to install
+
+* netcoreapp 3.1
+* MySql 8.0.21
+
+### Setup
+
+- clone this repo to your local machine using git clone
+- create an .env file to manage your application secrets:
+  https://github.com/tonerdo/dotnet-env
+ -create an email account for the notification service and write it to .env file
+ -use our scripts to create and fill the database with test data
```

Comment:
Maybe need to say 
"Add connection string for created database to .env file"

Reply:
Right, thanks